### PR TITLE
Hotfix user not returning list

### PR DESCRIPTION
**Pull request by DavidFair**
### Description:
Fixes UserRunner not returning a list, this was seen by @meoflynn 

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [x] Added unit tests?
* [ ] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

**Comment by anish-mudaraddi** [comment]
rewrite your tests once #141 is merged

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
rewrite your tests once #141 is merged

**Review by meoflynn — CHANGES_REQUESTED** [review]

File: lib/openstack_query/runners/user_runner.py [hunk 1]
```diff
@@ -91,6 +91,11 @@ def _run_query(
         if not filter_kwargs:
             filter_kwargs = {}
 
+        # having a filter of 'id' - will automatically mean no other filter kwargs
+        if "id" in filter_kwargs:
+            val = conn.identity.find_user("id", ignore_missing=True)
```
[the remark this answers is not in the document]
Comment (DavidFair) [reply lib/openstack_query/runners/user_runner.py:96]: Ah I flagged this last night too as a potential bug, I wanted to double check this morning but that confirms it. Thanks :+1:

**Review by meoflynn — APPROVED** [review]

**Review by anish-mudaraddi — APPROVED** [review]
LGTM

**Pull request merged**